dbms/src/Dictionaries/CacheDictionary.h

```diff
@@ -364,6 +369,9 @@ class CacheDictionary final : public IDictionary
 
         std::atomic<bool> is_done{false};
         std::exception_ptr current_exception{nullptr};
+
+        /// While UpdateUnit is alive, it is accounted in update_queue size.
+        CurrentMetrics::Increment alive_update_unit{CurrentMetrics::CacheDictionaryUpdateQueueSize};
```

Comment:
Why don't also add a metric with the number of keys?

Reply:
And we can rename the metrics accordingly (batches/keys).

Reply:
Will do.